entrypoint.sh

```diff
@@ -7,7 +7,7 @@ set +e
 git tag | xargs git tag -d
 for remote in `git branch -r`; do git branch --track ${remote#origin/} $remote; done
 git fetch --all --prune --tags
-git pull --all
+git pull --all --prune --tags
```

Comment:
Q: do we need fetch AND pull here?

Reply:
not sure, just did it because it works. Doesn't hurt IMHO. Do you think investigation and optimization worth it?